packages/tailwind/src/utils/config-handler.ts

```diff
@@ -57,15 +63,15 @@ const generateConfig = <T extends ThemeOptions>(theme: T, parsedColors: ParsedCo
 
   const colors = () =>
     ([] as StyleHandler[]).concat(
-      ...AVAILABLE_VARIANTS.map((variant) => toColorsStyles(parsedColors, variant))
+      ...AVAILABLE_VARIANTS.map((variant) => toColorsStyles(parsedColors, variant, prefixToAdd))
     );
 
   const sizes = () =>
     ([] as StyleHandler[]).concat(
       ([] as StyleHandler[]).concat(
         ...Object.keys(theme.sizes).map((size) => ({
           name: () => `${size}`,
-          styles: () => sanitize(theme.sizes[size])
+          styles: () => `${sanitize(theme.sizes[size])}`
```

Comment:
Not sure, but I don't think the "`" is needed here. I think is safe to remove even from line 73 as well.